packages/dashmate/src/docker/resolveDockerSocketPath.js

```diff
@@ -0,0 +1,21 @@
+const { promisify } = require('util');
+const path = require('path');
+const exec = promisify(require('child_process').exec);
+
+/**
+ * Returns docker socket path
+ * @returns {Promise<string|undefined>}
+ */
+async function resolveDockerSocketPath() {
```

Comment:
I wouldn't hide an error in the function but catch it on upper level where you use it. Please add a comment there that we skip error which is happening if docker is not installed or ran (and will be handled in logic bellow)